src/commands/ConfigurationFactory.h

```diff
@@ -60,7 +56,7 @@ class ConfigurationImpl : public Configuration {
 	 * Type definition for the hash_table that will map strings to
 	 * values in the configuration
 	 */
-	typedef hash_map<std::string, std::string, hash<std::string>, util::eqstr>
+	typedef unordered_map<std::string, std::string, hash<std::string>, util::eqstr>
```

Comment:
Good point, but it is not necessary because the "using namespace std" is specified in the giapi/Configuration.h file, and the ConfigurationFactory import it.

Reply:
I saw it used as `std::unorderde_map` and plain `unordered_map`. It maybe nice to use the same notation everwhyre